server/src/http/unleash_client.rs

```diff
@@ -538,19 +544,24 @@ impl UnleashClient {
         }
     }
 
-    pub async fn send_batch_metrics(&self, request: MetricsBatch) -> EdgeResult<()> {
+    pub async fn send_batch_metrics(
+        &self,
+        request: MetricsBatch,
```

Comment:
I expected a different type here (e.g. BatchMetricsRequest similarly to ClientFeaturesRequest in the fetch flags call), but since we have only MetricsBatch I am adding interval as a new param